indexer/src/parser.ts

```diff
@@ -28,6 +28,13 @@ const LanguageInfos: Array<LanguageInfo> = [
     maxLevel: 3,
     minLoc: 4,
   },
+  {
+    language: require("tree-sitter-typescript").typescript,
+    extensions: [".ts"],
+    nodeTypes: ["function_declaration", "class_declaration"],
+    maxLevel: 3,
```

Comment:
Have you verified the level settings? For now we only want to level function / class definitions.